backend/app/views/backoffice/projects/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<table class="backoffice-table">
+  <thead>
+    <tr>
+      <th>
+        <%= sort_link @q, "name_#{I18n.locale}", t("backoffice.common.project_name") %>
```

Comment:
Just two minor points:
 - maybe I would prepare wrapper for `sort_link` called for example `localized_sort_link`, because this will be probably required at other places as well
 - if I remember correctly how translation works so when `name_pt` is for example missing, it fallbacks to `name_en`, so theoretically you can see at this column text from selected language plus fallback one. This sorting will not work in such cases. On other hand, I don't think this is use case which will happen a lot so I would not spend much time with it. Just something to keep in mind :)

Reply:
Hmm, not sure how fallbacks work for translated column, if they actually do, but sorting by fallback we will need a `ransacker` helper I guess and complicate this more. I agree, I can add `localized_sort_link` helper.